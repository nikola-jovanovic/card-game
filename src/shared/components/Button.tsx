import styled from 'styled-components'

export default styled.button`
  display: block;
  width: 100%;
  margin-bottom: 5px;
  padding: 10px;
  border-radius: 8px;
  background-color: #e5f4e7;
  cursor: pointer;
  font-size: 16px;
`
