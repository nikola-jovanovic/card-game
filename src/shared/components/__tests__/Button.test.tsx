import { render, screen } from '@testing-library/react'
import React from 'react'

import Button from '../Button'

describe('Button', () => {
  it('render', () => {
    const { container } = render(<Button>click me</Button>)

    expect(screen.getByRole('button', { name: /click me/i })).toBeInTheDocument()
    expect(container).toMatchSnapshot()
  })
})