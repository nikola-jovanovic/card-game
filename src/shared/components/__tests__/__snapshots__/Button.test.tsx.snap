// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button render 1`] = `
.c0 {
  display: block;
  width: 100%;
  margin-bottom: 5px;
  padding: 10px;
  border-radius: 8px;
  background-color: #e5f4e7;
  cursor: pointer;
  font-size: 16px;
}

<div>
  <button
    class="c0"
  >
    click me
  </button>
</div>
`;
